src/signalesp.h

```diff
@@ -336,7 +341,8 @@ Serial.println("Starting config portal with SSID: NodeDuinoConfig");
 		}
 	}
 	*/
-wifiManager.autoConnect("NodeDuinoConfig",NULL);
+wifiManager.setConnectTimeout(20);
```

Comment:
Auch dieser Timeout ist zu knapp, bei meinen ESPs dauert das manchmal länger. Bei mir reichten bisher 60 Sekunden.

Dann würde ich noch folgendes hinzufügen:
`wifiManager.setConnectRetries(5); // max. 10`

Reply:
Die Timeouts habe ich angepasst.

`wifiManager.setConnectRetries(5); // max. 10`
Angenommen ich verende den Signalesp in einer andren Umgebung in der das konfigurierte WLAN nicht verfügbar ist, muss 5 Minuten gewartet werden bis man die Config vornehmen kann. Der erneute reconnect wäre durch die 2 Minuten Portal Timeout sowieso gegeben. Aus dem Grund würde die Retries eher nicht konfigureren.

Reply:
Stimmt auch wieder, da der ESP nach dem Timeout wifiManager.setConfigPortalTimeout(120) ja neu startet und dann wieder versucht sich anzumelden.